google-cloud-spanner/src/main/java/com/google/cloud/spanner/DatabaseInfo.java

```diff
@@ -63,11 +62,39 @@ public Builder setDialect(Dialect dialect) {
       throw new UnsupportedOperationException("Unimplemented");
     }
 
+    /**
+     * Optional for creating a new database.
+     *
+     * <p>The proto descriptors input as byte[] to be used for the database.
+     *
+     * <p>It is used by CREATE/ALTER PROTO BUNDLE statements which are part of DDL statements.
+     * Contains a protobuf-serialized [google.protobuf.FileDescriptorSet]. To generate a proto
+     * descriptors file run `protoc` with --include_imports and --descriptor_set_out.
+     */
     public abstract Builder setProtoDescriptors(@Nonnull byte[] protoDescriptors);
 
+    /**
+     * Optional for creating a new database.
+     *
+     * <p>The proto descriptors input as InputStream to be used for the database.
```

Comment:
nit: Please use javadoc tags while writing javadoc. Use `@param` for the parameters description.

Reply:
Done. Can you check it?